### PR TITLE
fixed(Label/Popup)

### PR DESCRIPTION
**Pull request by muskan2622**
### Functionality:
Delete Popup Label Fixed

### Solution:
Fixed Modal of Delete Popup

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
Go to my organisation and check

### CODE REVIEW

File: src/app/body/team-details/label-card/label-card.component.css [hunk 1]
```diff
@@ -11,21 +11,48 @@
...
     background-color: var(--secondary-bg);
 }
 
 .modal-content {
-    border: var(--primary-border);
-    background-color: var(--secondary-bg);
-    color: var(--text);
+  background-color: var(--card-bg);
+  color: var(--text);
+  border-radius: 6px;
```
Comment (viv9k) [inline src/app/body/team-details/label-card/label-card.component.css:27]: This should be 5 px

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required fix

File: src/app/body/team-details/label-card/label-card.component.css (hunk 1 above)
Comment (muskan2622) [reply src/app/body/team-details/label-card/label-card.component.css]: Done

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**